bedrock/firefox/views.py

```diff
@@ -541,6 +542,17 @@ def get_template_names(self):
         return [template]
 
 
+def fxos(request):
+    locale = l10n_utils.get_locale(request)
+    old_home = 'firefox/os/index.html'
+    new_home = 'firefox/os/index-new.html'
+
+    if lang_file_is_active(new_home, locale):
```

Comment:
Also, the confusion lang file vs template name is on me, given that I tested a code and wrote a different one in the bug comment.
```
if lang_file_is_active('firefox/os/index-new', locale):
```